pyEpiabm/pyEpiabm/routine/simulation.py

```diff
@@ -154,3 +170,4 @@ def set_random_seed(seed):
         """
         random.seed(seed)
         np.random.seed(seed)
+        logging.info(f"Set random seed to: {seed}")
```

Comment:
Any reason for doing this here, but not the other times random seed is set ?

Reply:
I've added it to the population seeds too